changes/ticket25549

```diff
@@ -0,0 +1,4 @@
+  o Minor features (continuous integration):
```

Comment:
This script still has some UTF-8 trouble, my name comes out sometimes quite strange.

Reply:
Hm. Any idea how to fix that?  I'm afraid I don't know.  If neither of us can figure it out, then we can open a ticket and hope somebody else gets it working.